.circleci/workflows.yml

```diff
@@ -110,23 +110,28 @@ jobs:
   install-npm-packages:
     executor: vm
     steps:
+      - attach_workspace:
+          at: /
+      - run:
+          name: Remove existing node versions
+          command: rm -rf /opt/circleci/.nvm/versions/node/*
       - restore_cache:
-          key: node18-packages-v3
+          key: node18-packages-v4
       - run:
-          name: Replace existing node versions with v18
+          name: Install node v18
           command: |
-            nvm deactivate
-            for ver in $(ls /opt/circleci/.nvm/versions/node/); do nvm uninstall ${ver}; done
             nvm install --default v18
             nvm version default > ~/.nvmrc
       - run: npm install prettier prettier-plugin-solidity@1.0.0-beta.19 --location=global
       - run: npm install solhint --location=global
       - run: npm install ganache --location=global
+      - run: echo 'nvm use v18' >> ~/.bashrc
       - save_cache:
-          key: node18-packages-v3
+          key: node18-packages-v4
```

Comment:
do we store the executables of the above installed packages here?

Reply:
> do we store the executables of the above installed packages here?

Yes.